site/docs/foundations/size.mdx

```diff
@@ -77,38 +79,66 @@ The purpose of `--salt-size-adornment` is to set the size of a discrete element
 
 <ImageSwitcher
   images={[
-    { src: "/img/foundations/size-adornment.svg", alt: "" },
-    { src: "/img/foundations/size-adornment-annotated.svg", alt: "" },
+    { src: "/img/foundations/size-adornment.png", alt: "" },
+    { src: "/img/foundations/size-adornment-annotated.png", alt: "" },
   ]}
   label="Show sizes"
 />
 
 ## Bar
 
-Bar is used to control the thickness of an accent bar, either in vertical or horizontal orientations.
+`--salt-size-bar` is used to define the thickness of an accent bar or track, suitable for both vertical and horizontal orientations, such as in [Card](../components/card) or [Progress](../components/progress).
 
 <ImageSwitcher
   images={[
-    { src: "/img/foundations/size-bar.svg", alt: "" },
-    { src: "/img/foundations/size-bar-annotated.svg", alt: "" },
+    { src: "/img/foundations/size-bar.png", alt: "" },
+    { src: "/img/foundations/size-bar-annotated.png", alt: "" },
   ]}
   label="Show sizes"
 />
 
-## Indicator
+### Bar-strong
 
-`--salt-size-indicator` controls the thickness of an indicator, like on Tabs or Navigation Item.
+`--salt-size-bar-strong` is used to control the thickness of an accent bar, when it is positioned on top of a track, such as in [Progress](../components/progress).
+
+<ImageSwitcher
+  images={[
+    { src: "/img/foundations/size-bar-strong.png", alt: "" },
+    { src: "/img/foundations/size-bar-strong-annotated.png", alt: "" },
+  ]}
+  label="Show sizes"
+/>
 
-## Border
+## Indicator
 
-Unlike other sizes, `--salt-size-border` is a fixed dimension regardless of density. It controls the size of all borders and separators across the design system.
+`--salt-size-indicator` defines the thickness of indicators, such as those used on tabs or navigation items.
```

Comment:
Should we change this to "such as those used in Tab or Navigation Item" with both components being links to the relevant pages.